### ISSUE TITLE
Admin-UI error when creating new User with byTracking()

### ISSUE THREAD

**Issue by srobrien** [keystonejs/keystone#2980]
Hello, not sure if this has been flagged but I noticed that when you set up a new project with an empty database and the first user is added using the api if the user model has the byTracking() plugin enabled, the admin-ui will throw a React error as soon as you log in. I assume it is trying to find a createdBy user but cannot locate one (obviously as the user created is the first one).  

See below: 

<img width="1277" alt="Screenshot 2020-05-17 at 18 40 02" src="https://user-images.githubusercontent.com/17919602/82155586-e3449000-986d-11ea-80e0-a8fa88fa27f4.png">

**Comment by MadeByMike** [comment]
@srobrien, I'm having trouble verifying the issue. Could I ask if it is possible to update all keystone packages in your repo, to rule out any clashes?

@gautamsi we checked this with our recent hook changes and I am using these in: https://github.com/keystonejs/keystone/pull/3044 I'm not aware of any issues are you?

**Comment by gautamsi** [comment]
This seems to be issue due to Flow, see error in screenshot with type information in SortLinkProps.
I am not facing this in any of my app. 

Hope updating all the dependencies fix it.

**Comment by komenixx** [comment]
@gautamsi Updating dependencies fix this issue but makes bug in Database (#3078). For me now is impossible to update packages.

**Comment by andyfcx** [comment]
![image](https://user-images.githubusercontent.com/11717383/83592835-d2925a80-a58d-11ea-923f-a948cc8a0660.png)
Had this problem too...

**Comment by MadeByMike** [comment]
Looks like this is definitely related to how people are updating dependencies. I've created a discussion here in case this can be improved through the right update process: https://github.com/keystonejs/keystone/discussions/3087 

Beyond that, I think we did the right things in terms of indicating breaking changes and dependencies between packages (but maybe we missed something). I don't want people to get errors when updating Keystone but I don't have any more of a solution.
